src/components/Form/Form.jsx

```diff
@@ -0,0 +1,53 @@
+import { useState } from "react";
+import styles from "./Form.module.scss";
+
+const Form = ({
+  searchTerm,
+  setSearchTerm,
+  setPrevSearchTerm,
+  isFetchingData,
+  setIsFetchingData,
+  setSearchResults,
+}) => {
+  const dataFetch = async () => {
+    const searchQuery = searchTerm.replaceAll(" ", "+");
+    const response =
+      await fetch(`https://www.googleapis.com/books/v1/volumes?q=${searchQuery}&maxResults=40
+`);
+    const results = await response.json();
+    const resultsToArray = Object.values(results);
+    const arrayOfBooks = resultsToArray[2];
```

Comment:
this could be a parametrised function defined somewhere else that takes in an argument
i.e 
```js
const searchForBooks = async (query) => { 
 const searchQuery = query.replaceAll(" ", "+");
    const response =
      await fetch(`https://www.googleapis.com/books/v1/volumes?q=${searchQuery}&maxResults=40
`);
    const results = await response.json();
    return results // or some function to process your results goes here 
    }
```